BUILDING.md

```diff
@@ -99,3 +98,11 @@ To find the Java binary in your environment, run the appropriate command for you
 * Linux/macOS: `which java`
 * Windows: `for %i in (java.exe) do @echo. %~$PATH:i`
 
+## Building documentation
+
+Required:
```

Comment:
These steps don't work on Ubuntu 18.04. I tried it with the WSL and also on an Ubuntu VM:
```bash
$ pip install -r requirements.txt
[...]
Successfully installed Jinja2-2.10.1 Markdown-3.1 MarkupSafe-1.1.1 PyYAML-5.1 Pygments-2.3.1 click-7.0 livereload-2.6.0 markdown-include-0.5.1 mkdocs-1.0.4 mkdocs-markdownextradata-plugin-0.0.5 mkdocs-material-4.0.1 pymdown-extensions-6.0 setuptools-41.0.1 six-1.12.0 tornado-6.0.2
```

```bash
$ mkdocs serve

Command 'mkdocs' not found, but can be installed with:

sudo apt install mkdocs
```

It might be that a newer version of `pip` is necessary as Ubuntu installs it in version 9.0.1 which is quite outdated and the installation instructions of Mkdocs [also mention updating pip](https://www.mkdocs.org/#installation).

Apart from that, please write a full sentence here to explain that this is required to install MkDocs with its requirements. Then I would write two more sentences to explain how the docs can be built and tested as that isn't part of the installation instructions (or _requirements_).